docs/platform/integration/magento2.md

```diff
@@ -136,8 +137,12 @@ If you're using `demo-v1` this will be set up to use our demo Magento 2 instance
 
 ### Deity Cloud Environment Variables
 
-- `PAYMENT_STRIPE_SECRET_KEY`: your secret key
-- `PAYMENT_STRIPE_PUBLIC_KEY`: your public key
+- `MAGENTO_SITE_URL_WITHOUT_PROTOCOL`: Your Magento site URL (without protocol) e.g. magento.deity.io
+- `CONSUMER_KEY`: your consumer key
+- `CONSUMER_SECRET`: your consumer secret
+- `ACCESS_TOKEN`: your access token
+- `ACCESS_TOKEN_SECRET`: your access token secret
+- `MAGENTO_DEFAULT_LOCALE`: default locale for your Magento store. e.g. `en_GB`
```

Comment:
@tutok @mateuszwozniak - what format does this expect?

Reply:
@hobadams could you please also add custom-env-vars to FP repo? currently, we don't have any [env vars for magento](https://github.com/deity-io/falcon-platform/blob/dev/examples/demo-v1/server/config/custom-environment-variables.json), so these vars won't be applied. 

And could we please use `MAGENTO_` prefix for these keys? (as there might be many components/APIs that could be using their own tokens)

Reply:
should we update it for multistore? If yes, then I do not thing so that we are using default locale, but @mateuszwozniak  will know better I think.
In terms of locale format it is following:

```
Language tag of the scope. This has to be in format of [language]-[region]. See https://en.wikipedia.org/wiki/Language_localisation#Language_tags_and_codes for reference
```